src/docs/cookbook-payments/removing-stripe/index.md

```diff
@@ -0,0 +1,332 @@
+    },
+-    {
+-      text: <FormattedMessage id="LayoutWrapperAccountSettingsSideNav.paymentsTabTitle" />,
+-      selected: currentTab === 'StripePayoutPage',
+-      id: 'StripePayoutPageTab',
+-      linkProps: {
+-        name: 'StripePayoutPage',
+-      },
+-    },
+-    {
+-      text: <FormattedMessage id="LayoutWrapperAccountSettingsSideNav.paymentMethodsTabTitle" />,
+-      selected: currentTab === 'PaymentMethodsPage',
+-      id: 'PaymentMethodsPageTab',
+-      linkProps: {
+-        name: 'PaymentMethodsPage',
+-      },
+-    },
+  ];
+
+  return <LayoutWrapperSideNav tabs={tabs} />;
+```
+
+Changes to `routeConfiguration.js`
+
+```diff
+-export const ACCOUNT_SETTINGS_PAGES = [
+-  'ContactDetailsPage',
+-  'PasswordChangePage',
+-  'StripePayoutPage',
+-  'PaymentMethodsPage',
+-];
++export const ACCOUNT_SETTINGS_PAGES = ['ContactDetailsPage', 'PasswordChangePage'];
+```
+
+### Extra: Removing unused exports
+
+This is not a mandatory step to do but if you want to clean up the code you can
+go through the `components/index.js`, `containers/index.js` and
+`forms/indexs.js`, and remove the Stripe components. After this, you should go
+through the whole codebase and remove all the references to these components. We
+have tried to follow the naming pattern where Stripe related components have
+`Stripe` in their name but there are a couple of exceptions like
+`PaymentMethodsPage`. All these components should be removed also from
+`examples.js`.
+
+Stripe related components:
+
+- StripeBankAccountTokenInputField
+- StripeConnectAccountStatusBox
+- StripePaymentAddress
+- StripePayoutPage
+- PaymentMethodsPage
+- PaymentMethodsForm
+- StripePaymentForm
+- StripeConnectAccountForm
+
+You can also remove Stripe script form
+[public/index.html](https://github.com/sharetribe/ftw-daily/blob/master/public/index.html#L169).
+
+# Removing payments
+
+If you want to remove payments completely from your marketplace, there are even
+more things to consider. Again it depends a lot on your use-case what kind of
+changes are needed. If you e.g. want to add the second transaction process with
+free transactions you need to build the conditional logic for showing pricing
+components when needed and hiding them when not. If you want to build the
+marketplace completely without payments then you most likely want to hide all
+the references to the listing price.
+
+### More changes to the transaction process
+
+Remove the actions related to payments e.g. in flex-default-process
+:action/privileged-set-line-items and :action/calculate-full-refund. In older
+processes this means actions related to calculating total price, commission etc.
+Read more about actions from
+[transaction process actions refrence article.](/references/transaction-process-actions/)
+Remember also to update email templates which contain pricing information.
+
+### Edit API calls in FTW backend
+
+If you are using FTW-daily v6.0.0 or higher or FTW-hourly v8.0.0 or higher you
```

Comment:
Lot's of "or" conjunctions that are connecting items a bit strangely. 

One alternative suggestion:

```suggestion
If you are using FTW-daily v6.0.0 or higher (or FTW-hourly v8.0.0 or higher) you
```